### PR TITLE
Models done with 100% test coverage

### PR DESCRIPTION
**Pull request by Iglesias-Leafwind**

### CODE REVIEW

File: src/main/java/pt/ua/tqs/fourwheels/models/CarModel.java [hunk 1]
```diff
@@ -1,23 +1,28 @@
...
     public Car getCar() {
         return car;
     }
 
     public void setCar(Car car) {
         this.car = car;
     }
+
+    @Override
+    public String toString() {
```
Comment (user-cube) [inline src/main/java/pt/ua/tqs/fourwheels/models/CarModel.java:23]: Do you really need a to string?

File: src/main/java/pt/ua/tqs/fourwheels/models/CarModel.java [hunk 2]
```diff
@@ -1,23 +1,28 @@
 package pt.ua.tqs.fourwheels.models;
 
-import lombok.AllArgsConstructor;
-import lombok.Data;
-import lombok.NoArgsConstructor;
-import lombok.ToString;
 import pt.ua.tqs.fourwheels.entities.Car;
 
-@Data
```
Comment (user-cube) [inline src/main/java/pt/ua/tqs/fourwheels/models/CarModel.java:9]: Finally lombok removed.

File: src/main/java/pt/ua/tqs/fourwheels/models/UserModel.java [hunk 3]
```diff
@@ -19,4 +22,12 @@ public String getPassword() {
     public void setPassword(String password) {
         this.password = password;
     }
+
+    @Override
+    public String toString() {
```
Comment (user-cube) [inline src/main/java/pt/ua/tqs/fourwheels/models/UserModel.java:27]: Do you really need this to String?

File: src/test/java/pt/ua/tqs/fourwheels/repositories/ProfileRepositoryTest.java [hunk 4]
```diff
@@ -8,7 +8,7 @@
 @WebMvcTest(value = ProfileRepositoryTest.class)
 class ProfileRepositoryTest {
     @Test
-    private void injectedComponentAreNotNullTest() {
+    public void injectedComponentAreNotNullTest() {
```
Comment (user-cube) [inline src/test/java/pt/ua/tqs/fourwheels/repositories/ProfileRepositoryTest.java:11]: Why do we need this piece of code? It's acctually making nothing.

File: src/main/java/pt/ua/tqs/fourwheels/models/CarModel.java (hunk 1 above)
Comment (joaolaranjo) [reply src/main/java/pt/ua/tqs/fourwheels/models/CarModel.java:23]: Solved! You were right!

File: src/main/java/pt/ua/tqs/fourwheels/models/CarModel.java (hunk 2 above)
Comment (joaolaranjo) [reply src/main/java/pt/ua/tqs/fourwheels/models/CarModel.java:9]: Deleted!

File: src/main/java/pt/ua/tqs/fourwheels/models/UserModel.java (hunk 3 above)
Comment (joaolaranjo) [reply src/main/java/pt/ua/tqs/fourwheels/models/UserModel.java:27]: Nope, we dont! Delete!

**Pull request merged**